spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -150,6 +150,23 @@ module RSpec::Core
           |     #   ./spec/rspec/core/formatters/exception_presenter_spec.rb:133
         EOS
       end
+
+      it "adds extra failure lines from the example metadata" do
+        extra_example = example.clone
+        failure_lines = 'http://www.example.com/job_details/123'
+        extra_example.metadata[:extra_failure_lines] = "\n#{failure_lines}"
+        the_presenter = Formatters::ExceptionPresenter.new(exception, extra_example, :indentation => 4)
+
+        expect(the_presenter.fully_formatted(1)).to eq(<<-EOS.gsub(/^ +\|/, ''))
+          |
+          |    1) Example
+          |       Failure/Error: # The failure happened here!#{ encoding_check }
+          |         Boom
+          |         Bam
+          |         #{failure_lines}
```

Comment:
Do you think the presenter should insert a blank line between the failure message and the extra failure lines if there are any?  Seems like that would look better.